lib/plugins/aws/provider.js

```diff
@@ -480,6 +480,16 @@ class AwsProvider {
             required: ['Fn::Sub'],
             additionalProperties: false,
           },
+          awsCfToJsonString: {
+            type: 'object',
+            properties: {
+              'Fn::ToJsonString': {
+                anyOf: [{ type: 'object' }],
```

Comment:
Let's drop `anyOf` wrapper (it doesn't make much sense to create `anyOf` with just one option)

Reply:
I think it actually needs to be 

```js
anyOf: [{ type: 'object' }, { type: 'array' }],
```

as both object an array are valid input. With only `{ type: 'object' }`, arrays aren't accepted.

I've adjusted it to that and tested with various `serverless.yaml` configs. Agree?

Reply:
Sure that's great. Please re-request review once you feel it's ready

Reply:
PR is updated